project.clj

```diff
@@ -140,8 +140,14 @@
                          {:dependencies [[clj-kondo "2021.03.31"]]}]
 
              :eastwood [:test
-                        {:plugins [[jonase/eastwood "0.4.0"]]
+                        {:plugins [[jonase/eastwood "0.9.1"]]
                          :eastwood {:config-files ["eastwood.clj"]
                                     :exclude-namespaces [cider.nrepl.middleware.test-filter-tests]
                                     :ignored-faults {:unused-ret-vals-in-try {cider.nrepl.middleware.profile-test [{:line 25}]}
+                                                     ;; This usage of `proxy` can't avoid reflection warnings given that the `proxy` construct dispatches based on name only:
+                                                     :reflection {cider.nrepl.middleware.out [{:line "55"}
```

Comment:
These strings should be numbers - it's a minor bug I just found out now

(will fix later today)

Reply:
> (will fix later today)

done && amended!